explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -91,6 +91,15 @@ def symbol_summary():  # pylint: disable=unused-variable
 		template_name, context = symbol_routes_facade.html_summary()
 		return render_template(template_name, **context)
 
+	def _get_json_nodes(role, exact_match, request_args):
+		only_ssl = True if 'only_ssl' in request_args else None
```

Comment:
Is there a reason you are not checking if ``only_ssl`` is true?  vs it is in the query list 🤔

Reply:
yes, as the variable is named `only_ssl`, we can consider the SSL to be true.

the query will be like this:

`/api/symbol/nodes/peer?only_ssl`

Reply:
Yes, but what if some use this without knowing the API?
``/api/symbol/nodes/peer?only_ssl=false``

Reply:
make sense. let me make some improvement for that.